src/pages/admin-ui-sdk/reference/page-app.md

```diff
@@ -1,6 +1,83 @@
+
+`PageApp` displays out-of-process pages in the Commerce Admin UI using an `iframe`. This component uses a single prop, extensionId, to retrieve the title and URL. The component uses the title to change the Admin UI page title, and the URL to load the page into its `iframe`.
+
+## PageApp functions
+
+| Name              | Args        | Type     | Description                                                         |
+| ----------------- | ----------- | -------- | ------------------------------------------------------------------- |
+| `updatePageTitle` | `pageTitle` | `string` | Sets the title of the Admin page to the title of the `iframe` page. |
+
+### `updatePageTitle` usage
+
+The `PageApp` component uses the `updatePageTitle` function internally to set the title of the Admin UI page to the title of the `iframe` page. But you can also use this function directly within your extension to change the Admin UI page title.
+
+```js
+// file.js within your extension
+
+import { updatePageTitle } from '@adobe/commerce-backend-ui-extensibility/components/PageApp';
+
+updatePageTitle('My Custom Title');
+```
+
+## PageApp props
+
+| Name          | Type     | Required | Default | Description                                     |
+| ------------- | -------- | -------- | ------- | ----------------------------------------------- |
+| `extensionId` | `string` | true     | null    | The id that identifies the out-of-process page. |
+
+## PageApp usage
+
+The following example shows how to use the `PageApp` component within your extension. The example assumes that you have already created an `extensionsProvider` object that contains the extension configuration. For more information, see [Creating an extensions provider](../extensions-provider.md).
+
+```js
+// index.js of your extension
+
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import { Extensible, createExtensionRegistryProvider } from '@adobe/uix-host-react'
+import { PageApp } from '@adobe/commerce-backend-ui-extensibility/components/PageApp';
+
+// Your extension's async init function: main.
+
+async function main(config) {
+  const extensionsProvider = () => fetch(config.registryUrl)
+    .then(providerConfig => providerConfig.json())
+    .then((config) => {
+      return createExtensionRegistryProvider(config)();
+    }).catch (e => {
+      console.error(e.message);
+      return () => []
+    });
+
+// If the extensionId is defined and the uix-page-content element exists, 
+// render the PageApp component, which returns an iframe that loads the page.
+
+if (config.extensionId !== undefined && !!document.getElementById('uix-page-content')) {
+  const contentElement = ReactDOM.createRoot(document.getElementById('uix-page-content'));
+  contentElement.render(
+    <React.StrictMode>
+      <Extensible extensionsProvider={extensionsProvider}>
+        <PageApp extensionId={config.extensionId}/>
+      </Extensible>
+    </React.StrictMode>
+  );
+}
+```
```

Comment:
The highlighting on GH gets weird here.  Maybe there's some stray non-rendering character nearby